gradio/blocks.py

```diff
@@ -433,7 +433,10 @@ def launch(
                 getpass.getpass("Enter key for encryption: ")
             )
 
-        self.enable_queue = enable_queue
+        if self.is_space and enable_queue is None:
+            self.enable_queue = True
+        else:
+            self.enable_queue = enable_queue
```

Comment:
What happens if `enable_queue` is not provided and you are *not* running in Spaces? Should explicitly make the default `False`

```suggestion
            self.enable_queue = enable_queue or False
```